src/core/plugin/luapi_application.h

```diff
@@ -1441,35 +1463,26 @@ static int applib_getToolInfo(lua_State* L) {
         Tool& tool = toolHandler->getTool(TOOL_TEXT);
         Color color = tool.getColor();
 
-        lua_newtable(L);
-
-        lua_pushliteral(L, "font");
-        lua_newtable(L);
```

Comment:
It looks to me that you removed one `lua_newtable(L)` too much (needed for the `font` table).

Reply:
Oh that's pretty important. Thanks for noting. I must have been mixing up `{font={size=0, name="fontname"}, color=0x}` and `{size=0, name="fontname", color=0x}` (leaving in a state where neither the first nor the second one is correctly implemented). Fixed it to the first one (+ clarified in comments) as this is the structure being used before.